doc/sphinx/addendum/type-classes.rst

```diff
@@ -138,36 +141,45 @@ the constraints as a binding context before the instance, e.g.:
 These instances are used just as well as lemmas in the instance hint
 database.
 
+.. _contexts:
+
 Sections and contexts
 ---------------------
 
-To ease the parametrization of developments by typeclasses, we provide a new
-way to introduce variables into section contexts, compatible with the implicit
-argument mechanism. The new command works similarly to the :cmd:`Variables`
-vernacular, except it accepts any binding context as argument. For example:
+To ease the parametrization of developments by typeclasses, one can use
+the command :cmd:`Context` to introduce variables into section contexts,
+it works similarly to the :cmd:`Variable` vernacular, except it accepts any
```

Comment:
The text here does sound strange. It is meant to be parsed as "parametrization (...) by typeclasses", but it might be more accurate to say "parameterization by instances". The intent is really to be able to write multiple definitions with the same parameters, some of which happen to be class instances.

Reply:
What is a "development"?  I don't get it.  Does it just mean something a user is working on?  If so, I don't see how the word is useful here.  Seems like "The intent is really to be able to write multiple definitions with the same parameters, some of which happen to be class instances." is getting closer to the point.

Maybe discard the sentence and start from scratch, maybe something like: "The Context command lets you define variables that are local to the current context.  Outside of the section, these variables become parameters. This is an easy way to write multiple definitions that use the same parameters (some of which may be class instances)."  (If that's accurate.)